src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -140,15 +140,15 @@
             </trans-unit>
             <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Deze waarde representeert geen geldige taal.</target>
+                <target>Deze waarde is geen geldige taal.</target>
             </trans-unit>
             <trans-unit id="39">
                 <source>This value is not a valid locale.</source>
-                <target>Deze waarde representeert geen geldige lokalisering.</target>
+                <target>Deze waarde is geen geldige locale.</target>
```

Comment:
If i throw this in google translate, I get "local" from Dutch "locale" to English. The dictionary doesn't disapprove it and recognizes it but has no explanation: http://www.woorden.org/woord/locale

However, the results are a odd, while it's technically correct, it feels wrong. I'd say locale is still better than it was before.